### DISCUSSION TITLE
Prisma $queryRaw is not working

### DISCUSSION THREAD

**Discussion by sunandini-chace** [prisma/prisma#19435] [Q&A]
I wanted to fetch all the Url  which match the  regex , For this I am using 
```
await prisma.$queryRaw(`
          SELECT *
          FROM links
          WHERE url ~ '^.*pazago.com\/track\?id=[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$'
        `);
```
        
  In pgAdmin  this query is giving the data but  I am getting empty array  here
  
I didn’t  understand  where I am doing wrong.

**Comment by nurul3101** [comment]
Hello @sunandini-chace 👋 

Is this happening on the latest version of Prisma i.e. `4.15.0`?

**Reply by nurul3101** [reply]
I am checking to see if you had a chance to look at my last response.
Please feel free to let us know in case you still need any assistance.

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have the a similar problem :pray:
